src/style.css

```
@@ -1,3 +1,74 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --active-color: rgba(0, 172, 220, 1);
+  --main-color: rgba(0, 0, 0, 1);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.logo_link {
+  height: 40px;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  min-width: 1024px;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav__link {
+  font-size: 12px;
+  line-height: 14px;
+  padding: 20px 5px;
```

only vertical padding needed
```suggestion
  padding: 23px 0;
```